charts/pgbouncer/templates/pgbouncer-deployment.yaml

```diff
@@ -0,0 +1,62 @@
+{{- if .Values.global.pgbouncer.enabled }}
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: {{ .Release.Name }}-pgbouncer
+  labels:
+    tier: astronomer
+    component: pgbouncer
+    release: {{ .Release.Name }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: 1
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+  selector:
+    matchLabels:
+      app: {{ template "pgbouncer.name" . }}
+      release: {{ .Release.Name }}
+      component: pgbouncer
+  template:
+    metadata:
+      labels:
+        app: {{ template "pgbouncer.name" . }}
+        chart: {{ template "pgbouncer.chart" . }}
+        heritage: {{ .Release.Service | quote }}
+        release: {{ .Release.Name }}
+        component: pgbouncer
```

Comment:
`spec.template.metadata.labels` now requires both `app` and `version`. see https://github.com/astronomer/astronomer/pull/1609